outputs/exec.go

```diff
@@ -0,0 +1,55 @@
+package outputs
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/aquasecurity/postee/layout"
+)
+
+type execCmd = func(string, ...string) *exec.Cmd
+
+type ExecClient struct {
+	ExecCmd   execCmd
+	Name      string
+	Env       []string
+	InputFile string
+	Output    []byte
+}
+
+func (e *ExecClient) GetName() string {
+	return e.Name
+}
+
+func (e *ExecClient) Init() error {
+	e.ExecCmd = exec.Command
+	e.Name = "Exec Output"
+	return nil
+}
+
+func (e *ExecClient) Send(m map[string]string) error {
+	e.Env = append(e.Env, os.Environ()...)
+	e.Env = append(e.Env, fmt.Sprintf("POSTEE_EVENT=%s", m["description"]))
```

Comment:
is `e.Env` cleared somewhere? Since single instance of output class is used

Reply:
Good catch - I've updated to now re-assign each time so we don't endlessly append like before https://github.com/aquasecurity/postee/pull/249/commits/1633f746010f435c789f3c04e245fae709b1b658